src/python/nimbusml/internal/core/feature_extraction/text/ngramfeaturizer.py

```diff
@@ -79,22 +79,7 @@ class NGramFeaturizer(BasePipelineItem, SingleOutputSignature):
         * ``"Spanish"``
         * ``"Japanese"``.
 
-    :param stop_words_remover: Specifies the stopwords remover to use. There
-        are
-        three options supported:
-
-        * `None`: No stopwords remover is used.
-        * :py:class:`PredefinedStopWordsRemover
-          <nimbusml.feature_extraction.text.stopwords.PredefinedStopWordsRemover>` :
-          A precompiled language-specific lists
-          of stop words is used that includes the most common words from
-          Microsoft Office.
-        * :py:class:`CustomStopWordsRemover
-          <nimbusml.feature_extraction.text.stopwords.CustomStopWordsRemover>` : A
-          user-defined list of stopwords. It accepts
-          the following option: ``stopword``.
-
-        The default value is `None`.
+    :param use_predefined_stop_word_remover: Use stop remover or not.
```

Comment:
>use_predefined_stop_word_remover [](start = 11, length = 32)

Are we breaking API here?

Reply:
can break before we ship 1.0

---
In reply to: [246972269](https://github.com/Microsoft/NimbusML/pull/77#discussion_r246972269) [](ancestors = 246972269)